source/utilities.tex

```diff
@@ -4845,8 +4845,10 @@
 
 \pnum
 \effects
-Let \tcode{is...} be \tcode{vars.index()...}. Returns \tcode{\placeholdernc{INVOKE}(forward<Visitor>(vis), get<is>(}\brk{}
-\tcode{forward<Variants>(vars))...);}\iref{func.require}.
+Let \tcode{is...} be \tcode{vars.index()...}. Returns:
+\begin{codeblock}
+@\placeholdernc{INVOKE}@(std::forward<Visitor>(vis), get<is>(std::forward<Variants>(vars))...); // see \ref{func.require}
```

Comment:
Please remove the semicolon. It has always been a mistake; we return expressions, not statements. Other instances of Returns: plus codeblock also never use a semicolon.

I could further question whether this shouldn't be a Returns: element rather than an Effects: one, but let's leave that as is for now.

Reply:
Done. I had similar thoughts about this being an odd use of `Effects`, and also decided it wasn't worth touching for now.

Reply:
That's fine, leave it with me. I'll discuss it with Richard.

Reply:
Isn't the semi-colon consistent with the third form at https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#writing-effects-in-a-function-description ?

Reply:
@jwakely: Only if we actually said `return`. But we don't. But I think we should change this to use a *Returns*: element anyway, non?